frontend/ui/screensaver.lua

```diff
@@ -192,6 +194,15 @@ function Screensaver:expandSpecial(message, fallback)
         end
         -- Unable to set time_left_chapter and time_left_document without ReaderUI, so leave N/A
     end
+    local powerd = Device:getPowerDevice()
```

Comment:
Could use tighter scoping.

(i.e., fold it inside the hasBattery check ;).)

Reply:
Fixed but hey man I'm just following your example ;)

https://github.com/koreader/koreader/blob/4a355f789d1f77218d892f3a9192e395419bb2a4/frontend/ui/widget/touchmenu.lua#L708-L719

Some others ones around too I think